public/app/plugins/datasource/cloudwatch/components/MetricStatEditor/MetricStatEditor.test.tsx

```diff
@@ -36,68 +35,68 @@ const props = {
   onRunQuery: jest.fn(),
 };
 
-afterEach(cleanup);
```

Comment:
Removed as this is not required.

https://testing-library.com/docs/react-testing-library/api/#cleanup

> Please note that this is done automatically if the testing framework you're using supports the `afterEach` global and it is injected to your testing environment (like mocha, Jest, and Jasmine). If not, you will need to do manual cleanups after each test.